test/JDBC/expected/switchoffset.out

```diff
@@ -0,0 +1,110 @@
+Select switchoffset('2001-04-22 ', -120)
```

Comment:
Let us add test case for all boundary values for date (min/max), time(min/max/0) and offset. For offset, boundary values are based on where we raise the error. Also, make sure to add some invalid values for date/offset, different data types.

Reply:
Added